src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityMenu.js

```diff
@@ -3,10 +3,10 @@ import * as React from 'react';
 import TetherComponent from 'react-tether';
 import { FormattedMessage } from 'react-intl';
 import DeleteConfirmation from '../common/delete-confirmation';
-import IconEdit from '../../../../icons/general/IconEdit';
 import IconTrash from '../../../../icons/general/IconTrash';
 import Media from '../../../../components/media';
 import messages from './messages';
+import Pencil16 from '../../../../icon/line/Pencil16';
```

Comment:
Can you rename it to `IconEdit`?

Reply:
Does it need to be `IconEdit`? In the existing implementation of `Comment.js I'm seeing that it is defined as `Pencil16`.

Reply:
If you search `Icon*` in BUIE codebase, you'll realize it's like a naming convention. And I think it's a good practice to follow naming conventions.

Reply:
Being a bit nosy :) The legacy icons (in `icons/general`) are being deprecated in favor of ones coming from `icon/line`, and `icon/fill` which come from our design asset import job. we should try to move off of those general icons, and use the equivalents in the aforementioned directories instead.

The new naming convention is icon/{type}/{ImageName}{baseResolution}.tsx

Lastly, we could move the trash icon from the `/general` version to the line-equivalent here, to keep consistency. Any styling should be applied via SASS or styled components (prefer sass since most colors are unchanging at runtime).